### ISSUE TITLE
TexturePacker error

### ISSUE THREAD

**Issue by abbaselmas** [BigNoid/Aeon-Nox#659]
TexturePacker give error because of the blank spaces on file names in flags/studios

**Comment by BigNoid** [comment]
Logfile or it didnt happen.

**Comment by abbaselmas** [comment]
You are right. I am trying to be sure at first. I will come back with log

**Comment by abbaselmas** [comment]
Ok here is the thing that I have done.

I have completely replace confluence with Aeon Nox and remove confluence. In the confluence media directory there exist a Makefile which dont include much. I have copied that Makefile to Aeon Nox media directory. In the kodi main Makefile chane the required confluence to Aeon Nox changes.

At the stage of building kodi (~/kodi-android make) no rule to make target error appear at Aeon Nox media/flags directory. 

Especially at studios directory giving 
no rule to make target Walt
no rule to make target Disney
no rule to make target Digital
no rule to make target Film
.
.
Removing blank spaces resolve the issue and compiled successfully as a default theme. (I have changed the blank spaces with these command btw: find -name "* *" -type f | rename 's/ /_/g')

maybe this is not an issue on your theme but my compile environment issue after all. Just wanna say
Thanks for your theme :)
